diff --git a/node_modules/react-native-smtp-mailer/android/build.gradle b/node_modules/react-native-smtp-mailer/android/build.gradle
index 822b5a0..ff2eb61 100644
--- a/node_modules/react-native-smtp-mailer/android/build.gradle
+++ b/node_modules/react-native-smtp-mailer/android/build.gradle
@@ -32,6 +32,9 @@ repositories {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation fileTree(dir: 'libs', include: '*.jar')
+    // implementation fileTree(dir: 'libs', include: '*.jar')
+
+    implementation 'com.sun.mail:android-mail:1.6.6'
+    implementation 'com.sun.mail:android-activation:1.6.6'
 }
   
\ No newline at end of file
